mailpoet/lib/WPCOM/DotcomHelperFunctions.php

```diff
@@ -48,6 +48,40 @@ public function isEcommerce(): bool {
     return function_exists('wc_calypso_bridge_is_ecommerce_plan') && wc_calypso_bridge_is_ecommerce_plan();
   }
 
+  public function isGarden(): bool {
+    if (!function_exists('is_blog_garden')) {
+      return false;
+    }
+    $blog_id = \get_current_blog_id();
+    $site = function_exists('get_site')
+      ? \get_site($blog_id)
+      : (function_exists('get_blog_details') ? \get_blog_details($blog_id) : null);
+    return $site ? \is_blog_garden($site) : false;
+  }
+
+  protected function getSiteMetaValue(string $meta_key): ?string {
+    if (!function_exists('get_site_meta')) {
+      return null;
+    }
+    $blog_id = \get_current_blog_id();
+    $value = \get_site_meta($blog_id, $meta_key, true);
+    return is_string($value) && $value !== '' ? $value : null;
```

Comment:
```suggestion
    return empty($value) ?  null : $value;
```
Since the return type of [get_site_meta](https://developer.wordpress.org/reference/functions/get_site_meta/) can be mixed. Why not use empty?

Reply:
Thank you @triple0t for your suggestion. It makes sense, and I used your version. Quick question: how did you do that? I mean, how did you suggest a change that I could simply commit here on GitHub?

Reply:
After the build failed, I know why I used my original version. It is because we don’t want to return `mixed` from this function; we want a string. So I am returning my original version.

Reply:
> Quick question: how did you do that? I mean, how did you suggest a change that I could simply commit here on GitHub?

This is done with the "add suggestion" command
<img width="220" height="113" alt="Screenshot 2025-09-09 at 11 45 06 AM" src="https://github.com/user-attachments/assets/8970dc08-e1e0-4580-9716-4f6a8f9bac06" />

More info: https://egghead.io/lessons/github-add-suggestions-in-a-github-pr-review